oled.c

```diff
@@ -82,11 +82,12 @@ static bool is_debug_mode = 0;
 inline void SPISend(uint32_t base, uint8_t *data, int len)
 {
 	int i;
-	delay(400);
+	delay(1);
 	for (i = 0; i < len; i++) {
 		spi_send(base, data[i]);
 	}
-	delay(800);
+	while (!(SPI_SR(spi) & SPI_SR_TXE));
+	while ((SPI_SR(spi) & SPI_SR_BSY));
```

Comment:
Yes!  Not sure what happened here. I just did a rebase on current master and I'm sure that branch worked.
I'll look into it and fix it.